inst/MFTC_calculator/generate_parameters.R

```diff
@@ -1,9 +1,15 @@
+# this is to be used and run with each new EFU
+
 # Split up a TAWA parameters file into several TAWA parameter files,
 # one for each Tax Year
+
+# before running: set working directory to MFTC_calculator and create a folder
+# named TAWA_Parameters
+
 IncomeExplorer::split_TAWA_parameters(
```

Comment:
Did this work or did you have to source the R files manually?

Reply:
I had to manually create the folder and enter a directory to BEFU22